src/inmanta/app.py

```diff
@@ -425,6 +425,19 @@ def export_parser_config(parser: ArgumentParser) -> None:
         default=None,
     )
 
+    parser.add_argument(
+        "--compile-json",
+        dest="compile_json",
+        help="Export structured json containing compile data such as occurred errors.",
+        action="store_true",
+        default=False,
+    )
+    parser.add_argument(
+        "--compile-json-file",
+        dest="compile_json_file",
+        help="File to export compile json to. If omitted %s is used." % compiler.config.default_json_file,
+    )
```

Comment:
- Good point. I'm not sure however what would be a better name. For `inmanta compile` it's just `--json` and `--json-file` respectively, but `inmanta export` already has a json-related option so I tought it would be confusing to just use `--json` and `--json-file`. Perhaps `--export-compile-data` might've been better but then that might be confused with export itself.
- I'm not sure what conventions are. On the one hand you're right, on the other you might want to specify a json needs to be generated without overriding the default file. This sort of user-oriented stuff is not really my area of expertise. In any case, for this PR I'd argue for leaving both options, since they already exist for `inmanta compile`. If you think the options need reevaluation we could create a seperate ticket.

Reply:
* Personally, I like the name `--export-compile-data` more. But that's just an opinion of course.
* Indeed. It's better to keep things consistent. The `compile` command already has both options, so we should do the same here.

Reply:
I actually agree with you on the first point, but also in this case (semi-)consistency is probably more important, right?